cli/src/template.pest

```diff
@@ -19,10 +19,11 @@
 
 whitespace = _{ " " | "\t" | "\r" | "\n" | "\x0c" }
 
+ansi_escape = @{ "\\" ~ ("e" | "x1b" | "x1B") }
 string_escape = @{ "\\" ~ ("t" | "r" | "n" | "0" | "\"" | "\\") }
 string_content_char = @{ !("\"" | "\\") ~ ANY }
 string_content = @{ string_content_char+ }
-string_literal = ${ "\"" ~ (string_content | string_escape)* ~ "\"" }
+string_literal = ${ "\"" ~ (string_content | string_escape | ansi_escape)* ~ "\"" }
```

Comment:
Oh, I meant adding `\xNN` instead of `\e` (which isn't so common.) I'm not against adding `\e` though.

If we add `\e`, it doesn't have to be a separate `ansi_escape` rule. Just add `"e"` to `string_escape`.

Reply:
Ah, done.

Reply:
I just chucked the hex bytes in string_escape as well, let me know if you want them declared separately (i.e., as part of string_content or string_literal).